packages/create-svelte/bin.js

```diff
@@ -1,187 +1,167 @@
-				type: 'confirm',
-				name: 'value',
-				message: 'Directory not empty. Continue?',
-				initial: false
-			});
-
-			if (!response.value) {
-				process.exit(1);
-			}
-		}
+	if (dir) {
+		cwd = /** @type {string} */ (dir);
 	}
+}
 
-	const options = /** @type {import('./types/internal').Options} */ (
-		await prompts(
-			[
-				{
-					type: 'select',
-					name: 'template',
-					message: 'Which Svelte app template?',
-					choices: fs.readdirSync(dist('templates')).map((dir) => {
-						const meta_file = dist(`templates/${dir}/meta.json`);
-						const { title, description } = JSON.parse(fs.readFileSync(meta_file, 'utf8'));
-
-						return {
-							title,
-							description,
-							value: dir
-						};
-					})
-				},
-				{
-					type: 'select',
-					name: 'types',
-					message: 'Add type checking with TypeScript?',
-					initial: false,
-					choices: [
-						{
-							title: 'Yes, using JavaScript with JSDoc comments',
-							value: 'checkjs'
-						},
-						{
-							title: 'Yes, using TypeScript syntax',
-							value: 'typescript'
-						},
-						{ title: 'No', value: null }
-					]
-				},
-				{
-					type: 'toggle',
-					name: 'eslint',
-					message: 'Add ESLint for code linting?',
-					initial: false,
-					active: 'Yes',
-					inactive: 'No'
-				},
-				{
-					type: 'toggle',
-					name: 'prettier',
-					message: 'Add Prettier for code formatting?',
-					initial: false,
-					active: 'Yes',
-					inactive: 'No'
-				},
-				{
-					type: 'toggle',
-					name: 'playwright',
-					message: 'Add Playwright for browser testing?',
-					initial: false,
-					active: 'Yes',
-					inactive: 'No'
-				},
-				{
-					type: 'toggle',
-					name: 'vitest',
-					message: 'Add Vitest for unit testing?',
-					initial: false,
-					active: 'Yes',
-					inactive: 'No'
-				}
-			],
-			{
-				onCancel: () => {
-					process.exit(1);
-				}
-			}
-		)
-	);
-
-	options.name = path.basename(path.resolve(cwd));
-
-	await create(cwd, options);
-
-	console.log(bold(green('\nYour project is ready!')));
-
-	if (options.types === 'typescript') {
-		console.log(bold('✔ Typescript'));
-		console.log('  Inside Svelte components, use <script lang="ts">');
-	} else if (options.types === 'checkjs') {
-		console.log(bold('✔ Type-checked JavaScript'));
-		console.log('  https://www.typescriptlang.org/tsconfig#checkJs');
-	} else if (options.template === 'skeletonlib') {
-		const warning = yellow('▲');
-		console.log(
-			`${warning} You chose to not add type checking, but TypeScript will still be installed in order to generate type definitions when building the library`
-		);
-	}
+if (fs.existsSync(cwd)) {
+	if (fs.readdirSync(cwd).length > 0) {
+		const force = await p.confirm({
+			message: 'Directory not empty. Continue?',
+			initialValue: false
+		});
 
-	if (options.eslint) {
-		console.log(bold('✔ ESLint'));
-		console.log(cyan('  https://github.com/sveltejs/eslint-plugin-svelte3'));
+		if (force !== true) {
```

Comment:
What else besides a boolean can this be? If nothing, then we can simplify this
```suggestion
		if (!force) {
```

Reply:
The API is somewhat unfortunate — it can be `true` or `false` or a `Symbol` that indicates the user cancelled with Ctrl-C (or whatever the windows equivalent is). In this case we need to bail if `force` is `false` or the `Symbol`; this is the easiest way to do that. We could add a comment to make it clearer

```suggestion
		// bail if `force` is `false` or the user cancelled with Ctrl-C
		if (force !== true) {
```